src/__tests__/videoCard.test.js

```diff
@@ -0,0 +1,52 @@
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+
+import VideoCard from '../components/VideoCard';
+import VideoContext from '../context/videoContext';
+
+describe('VideoCard component tests', () => {
+  beforeEach(() => {
+    render(
+      <VideoContext>
+        <VideoCard
+          image={video.url}
+          title={video.title}
+          description={video.description}
+          createdAt={video.publishedAt}
+        />
+      </VideoContext>
+    );
+  });
+  const video = {
+    url: 'https://i.ytimg.com/vi/nmXMgqjQzls/mqdefault.jpg',
+    title: 'Video Tour | Welcome to Wizeline Guadalajara',
+    description:
+      'Follow Hector Padilla, Wizeline Director of Engineering, for a lively tour of our office. In 2018, Wizeline opened its stunning new office in Guadalajara, Jalisco, ...',
+    publishedAt: '2014-09-27',
+  };
+
+  test('should containt a card container', () => {
+    const card = screen.getByTestId(/card_container/i);
+    expect(card).toBeInTheDocument();
+  });
+
+  test('should containt a header image', () => {
+    const cardImage = screen.getByTestId(/img_header/i);
+    expect(cardImage.src).toBe(video.url);
+  });
+
+  test('should containt a card title', () => {
+    const cardTitle = screen.queryByText(video.title);
+    expect(cardTitle).toBeInTheDocument();
+  });
+
+  test('should containt a card description', () => {
+    const cardDescription = screen.queryByText(video.description);
+    expect(cardDescription).toBeInTheDocument();
+  });
+
+  test('should containt a video creation date', () => {
+    const videoCreation = screen.queryByText(video.publishedAt);
+    expect(videoCreation).toBeInTheDocument();
+  });
+});
```

Comment:
very good test!

As minor comment, you could move the `const video` at the top of the `describe` in order to improve readability.

This actually works because of JS hoisting.